css/index.css

```diff
@@ -143,4 +143,155 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+header{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin: 10px;
+    padding: 10px;
+    align-items: center
+}
+
+.nav-container {
+    display: flex;
+    justify-content: space-around;
+}
+
+.logo{
+ display: flex;
+ align-items: center;
+ justify-content: center
+}
+
+nav {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: flex-end;
+    margin-top: 20px;
+}
+
+nav a{
+    padding: 0px 40px;
+    text-decoration: none;
+    color: lightslategrey
+}
+
+.banner-container {
+    display: flex;
+    justify-content: flex-start;
+    margin: 30px 0px
+}
+
+.middle-content .boxes .box-1{
+    background-color: #008080
+}
```

Comment:
In some cases you have a new line between a class and another, in some others you don't.
Even if this is not something technically important it makes your project look sloppier. 
Try to remain consistent.